routers/user/auth.go

```diff
@@ -949,7 +949,7 @@ func LinkAccountPostRegister(ctx *context.Context, cpt *captcha.Captcha, form au
 		Name:        form.UserName,
 		Email:       form.Email,
 		Passwd:      form.Password,
-		IsActive:    !setting.Service.RegisterEmailConfirm,
+		IsActive:    !setting.Service.RegisterEmailConfirm || !setting.Service.RegisterManualConfirm,
```

Comment:
This will break `RegisterEmailConfirm` feature with `RegisterManualConfirm` disabled

Reply:
My bad. Is
```
IsActive:    !(setting.Service.RegisterEmailConfirm || setting.Service.RegisterManualConfirm),
```
better? I think it should work already.